layouts/shortcodes/dbm/postgres/self-hosted-install-kubernetes.en.md

```diff
@@ -0,0 +1,185 @@
+            configDataMap:
+              postgres.yaml: |-
+                cluster_check: true
+                init_config:
+                instances:
+                - host: <HOST>
+                  port: 5432
+                  username: datadog
+                  password: 'ENC[datadog_user_database_password]'
+                  dbm: true
+    ```
+
+    **Note**: For Postgres 9.6, add the following settings to the instance config where host and port are specified:
+
+    ```yaml
+    pg_stat_statements_view: datadog.pg_stat_statements()
+    pg_stat_activity_view: datadog.pg_stat_activity()
+    ```
+
+2. Apply the changes to the Datadog Operator with the following command:
+
+    ```shell
+    kubectl apply -f datadog-agent.yaml
+    ```
+  
+### Helm
+
+Using [Kubernetes and Integration | Helm][4] as a reference, below is how you would setup the Postgres Integration:
+
+1. Update your YAML configuration file (`datadog-values.yaml` in the Cluster Agent installation instructions) to include the following:
+
+    ```yaml
+    datadog:
+      clusterChecks:
+        enabled: true
+
+    clusterChecksRunner:
+      enabled: true
+
+    clusterAgent:
+      enabled: true
+      confd:
+        postgres.yaml: |-
+          cluster_check: true
+          init_config:
+          instances:
+          - dbm: true
+            host: <HOST>
+            port: 5432
+            username: datadog
+            password: 'ENC[datadog_user_database_password]'
+
+    ```
+
+    **Note**: For Postgres 9.6, add the following settings to the instance config where host and port are specified:
+
+    ```yaml
+    pg_stat_statements_view: datadog.pg_stat_statements()
+    pg_stat_activity_view: datadog.pg_stat_activity()
+    ```
+
+2. Deploy the Agent with the above configuration file from the command line:
+
+    ```shell
+    helm install datadog-agent -f datadog-values.yaml datadog/datadog
+    ```
+
+<div class="alert alert-info">
+For Windows, append <code>--set targetSystem=windows</code> to the <code>helm install</code> command.
+</div>
+
+### Configure with mounted files
+
+To configure a cluster check with a mounted configuration file, mount the configuration file in the Cluster Agent container at the path: `/conf.d/postgres.yaml`:
+
+```yaml
+cluster_check: true  # Make sure to include this flag
+init_config:
+instances:
+  - dbm: true
+    host: '<HOST>'
+    port: 5432
+    username: datadog
+    password: 'ENC[datadog_user_database_password]'
+
+    ## Required: For Postgres 9.6, uncomment these lines to use the functions created in the setup
+    # pg_stat_statements_view: datadog.pg_stat_statements()
+    # pg_stat_activity_view: datadog.pg_stat_activity()
+```
+
+### Configure with Kubernetes service annotations
+
+Rather than mounting a file, you can declare the instance configuration as a Kubernetes Service. To configure this check for an Agent running on Kubernetes, create a Service in the same namespace as the Datadog Cluster Agent:
```

Comment:
```suggestion
Instead of mounting a file, you can declare the instance configuration as a Kubernetes Service. To configure this check for an Agent running on Kubernetes, create a Service in the same namespace as the Datadog Cluster Agent:
```